pom.xml

```diff
@@ -49,6 +48,68 @@
 			<version>${hapifhir_version}</version>
 		</dependency>
 
+		<!-- spring boot security, one for app one for test -->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-security</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.security</groupId>
+			<artifactId>spring-security-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<!-- embedded h2 database -->
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<scope>runtime</scope>
+		</dependency>
+
+		<!-- MySQL database-->
+		<dependency>
+			<groupId>mysql</groupId>
+			<artifactId>mysql-connector-java</artifactId>
+			<scope>runtime</scope>
```

Comment:
Is this dependency required? As I noticed, no mysql used in the project, but h2.

Reply:
We use h2 for protocol and will change to mysql if necessary, sure we can comment this anyway

Reply:
I removed this. We can of course add it back in if we need it